Description: fix segfault when using inject-mode
             with no arguments. (Closes: 386150)
Author: Deniz Adrian <adrian@netzquadrat.de>
Last-Update: 2006-09-05
Index: packit-1.0/src/main.c
===================================================================
--- packit-1.0.orig/src/main.c
+++ packit-1.0/src/main.c
@@ -214,6 +214,8 @@ parse_inject_options(int argc, char *arg
         }
     }
 
+print_usage();
+
 parse_inject:
 #ifdef DEBUG
     fprintf(stdout, "DEBUG: parse_inject\n");
